kvmagent/kvmagent/test/shareblock_testsuite/test_sharedblock_ha.py

```
@@ -0,0 +1,149 @@
+
+        cls.origin_json_dump_get = http.json_dump_get
+        cls.origin_json_dump_post = http.json_dump_post
+        # 1 means storage is good, 2 means storage is failed, -1 means no way to check.
+        cls.condition_dict = {1: "success", 0: "fail", -1:"no_way"}
+        cls.fencer_result_dict = {1: "trigger", 0:"no_trigger"}
+        cls.sanlock_io_timeout = 5
+        cls.management_network_ok = True
+        cls.fencer_triggered = 0
+
+    def mock_zsblk_agent(self):
+        def mock_func(uri, body=None, headers={}, fail_soon=False, print_curl=False):
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)